contact.html

```diff
@@ -184,26 +184,38 @@ <h1 style="text-align:center">Contact Us</h1>
                       <!-- Phone -->
                       <div class="col-sm-4 address-group" style="margin-bottom: 30px;">
                         <span style="text-align: center">Phone</span>
-                        <a href="#" style="text-align: center">+ 123 4567 890</a>
-                        <a href="#" style="text-align: center">+ 123 7654 098</a>
+                        <a href="#" style="text-align: center">(702) 367-7133</a>
                       </div>
 
                       <!-- Address -->
                       <div class="col-sm-4 address-group" style="margin-bottom: 30px;">
                         <span style="text-align: center">Address</span>
-                        <p style="text-align: center">1200 some street, il, US</p>
+                        <p style="text-align: center">7884 W Sahara Ave #100</p>
+                        <p style="text-align: center">Las Vegas, NV 89117</p>
                       </div>
 
                       <!-- Hours -->
                       <div class="col-sm-4 address-group" style="margin-bottom: 30px;">
                         <span style="text-align: center">Open Hours</span>
                         <div style="max-width: 150px; margin: 0 auto;">
-                          <div style="display: inline-block">Mon-Fri:</div>
-                          <div style="float: right;">9am-5pm</div>
+                          <div style="display: inline-block">Mon-Wed:</div>
+                          <div style="float: right;">7am-6pm</div>
+                        </div>
```

Comment:
I'd prefer if Monday, Tuesday, and Wednesday were all on separate lines, like the Hours in the About section of the Home page